datamad2/models/users.py

```diff
@@ -30,6 +30,18 @@ def __str__(self):
         return f"{self.name}"
 
 
+class Subtask(models.Model):
+    data_centre = models.ForeignKey(to=DataCentre, on_delete=models.PROTECT, null=True, blank=True)
+    name = models.CharField(max_length=200, blank=True, null=True)
+    schedule_time = models.IntegerField(blank=True, null=True, help_text='Time in weeks to at which to schedule sub-task in reference to the reference time.',
```

Comment:
Does this work with negative weeks?
If so, would be good to mention that in the help text

Reply:
Tested and this does work. Updated in a commit.